pkg/util/types/types.go

```diff
@@ -131,3 +131,16 @@ func (s Subnets) String() string {
 	}
 	return out
 }
+
+func Difference(a, b AWSStringSlice) (ab AWSStringSlice) {
+	mb := map[string]bool{}
+	for _, x := range b {
+		mb[*x] = true
+	}
+	for _, x := range a {
```

Comment:
shouldn't this just be `=` instead of `:=`?

Reply:
`x` is local to the scope of the `for` so `:=` is appropriate.